Library/Formula/objective-caml.rb

```diff
@@ -2,14 +2,17 @@ class ObjectiveCaml < Formula
   desc "General purpose programming language in the ML family"
   homepage "http://ocaml.org"
   url "http://caml.inria.fr/pub/distrib/ocaml-4.02/ocaml-4.02.1.tar.gz"
-  sha1 "6af8c67f2badece81d8e1d1ce70568a16e42313e"
+  sha256 "3cbc7af5a3886c8c5af8dab5568d6256a191d89ecbd4aea18eaf5b47034c6138"
   revision 2
 
   head "http://caml.inria.fr/svn/ocaml/trunk", :using => :svn
 
   option "with-x11", "Install with the Graphics module"
+
   depends_on :x11 => :optional
 
+  fails_with :clang
```

Comment:
It's a mistake, sorry. Working on too many formula at the same time. We re-commit.